components/Slider/tests/unit/SliderTests.m

```diff
@@ -913,6 +913,42 @@ - (void)testThumbElevationDefault {
   XCTAssertEqual(self.slider.thumbElevation, MDCShadowElevationNone);
 }
 
+- (void)testThumbRadiusForState {
```

Comment:
Please add a test that the legacy behavior is not affected when the API is off.

Reply:
I believe this is the same as the comment above.

Reply:
I don't see the test added. Can you please clarify what you mean by "the same comment above"? Do you mean "Done"?

![screen shot 2018-08-27 at 3 40 21 pm](https://user-images.githubusercontent.com/1753199/44681759-88bb2e80-aa0f-11e8-9c27-fa214e397f1b.png)

Reply:
What's the difference between the legacy behavior and fallback behavior?

Reply:
Sorry, by "legacy behavior" I meant when `.statefulAPIEnabled = NO;`.  Fallback is that each of the UIControl states will use the value for `.normal` if no explicit value is configured for that state.

Reply:
Done